feature-libs/product-configurator/rulebased/occ/variant/converters/occ-configurator-variant-normalizer.ts

```diff
@@ -34,6 +34,12 @@ export class OccConfiguratorVariantNormalizer
       productCode: source.rootProduct,
       groups: [],
       flatGroups: [],
+      kbKey: {
```

Comment:
only initialize kbkey, if source.kbkey is defined?

Reply:
Agree. Done!